executor/src/main/resources/spark_intp.py

```diff
@@ -21,20 +21,10 @@
 import os
 import sys
 import zipimport
+sys.path.append(os.getcwd())
 from runtime import AmaContext, Environment
 
-# os.chdir(os.getcwd() + '/build/resources/test/')
```

Comment:
Bring back the comments for testing
add the sys.path.append to the spark_intp test